README.md

```diff
@@ -1,2 +1,218 @@
-# HelloMicroverse
-Project for module 1 day 2
+<!-- Project for module 1 day 2 -->
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents 
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HelloMicroverse] <a name="about-project"></a>
+
+> This project is about a excercise to understand Microverse tools
```

Comment:
Please remove these comments from your readme file🙏. These comments are there to guide you on how to personalize the readme template and should not be present in your final readme. Kindly remove all lines which have a comments that start like this

Reply:
Are the comments the lines that start whit ">" char?